examples/gfl-local.html

```diff
@@ -26,10 +26,7 @@
   window.addEventListener('load', function () {
     window.tido = new window.Tido({
       manifest: "http://localhost:8181/gfl/textapi/Z_1819-06-03_l/manifest.json",
-      labels: {
```

Comment:
I think here is also a misunderstanding. The labels section should not be removed completely since it can have other dependencies. Did you check the metadata? I think it was used there. In this PR only new translation keys should be added.

Reply:
Okay, I understand, I have added a latest commit to address this.